src/CMakeLists.txt

```diff
@@ -30,18 +30,23 @@ set( jarsDownloadDirectory jars )
 set( JAR_BUILD_TREE_LOCATION
   ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/${jarsDownloadDirectory}/
   )
-set( JAR_INSTALL_TREE_LOCATION
+set( JAR_INSTALL_LOCATION
   ${SCIFIO_INSTALL_LIBRARY_DIR}/${jarsDownloadDirectory}/
   )
-
+set( JAR_INSTALL_TREE_LOCATION
```

Comment:
JAR_INSTALL_TREE_LOCATION & JRE_BUILD_TREE_LOCATION are used here: https://github.com/InsightSoftwareConsortium/scifio-imageio/blob/master/src/itkSCIFIOImageIO.cxx.in#L300-L327

Is this what you are asking?